pkg/interface/src/register-sw.js

```diff
@@ -1,6 +1,5 @@
 
-
-if ("serviceWorker" in navigator) {
+if ("serviceWorker" in navigator && process.env.NODE_ENV === 'production') {
```

Comment:
Actually, I wonder — will this proc on a build?

Reply:
If I console.log NODE_ENV I do get `development` from within the file, but in prod we aren't in a node session, are we?